VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -253,6 +255,8 @@ def do_parse_context(self, operation, seqNo):
             self.log("waagent new path is used")
         if not _context:
             self.log("maybe no new settings file found")
+            if(self.eventlogger is not None):
+                self.eventlogger.dispose()
```

Comment:
this can be moved to a helper method with a try-catch to suppress failure?

Reply:
discussed offline.